build.xml

```diff
@@ -153,25 +153,52 @@
         <equals arg1="${ant.java.version}" arg2="1.8"/>
     </condition>
     <condition property="java.version.11">
-        <not><isset property="java.version.8"/></not>
+        <equals arg1="${ant.java.version}" arg2="11"/>
+    </condition>
+    <condition property="java.version.17">
```

Comment:
why do we add now 17 to build xml?
? It can't even compile with it as Benjamin is not ready with the UDF refactoring.... Also, I already had the changes to make It 11 and 17, double work

Reply:
We don't need to do as complete a job as you, as we want to get JDK17 (actually JDK19) supported for the simulator only.